documentation/docs/03-loading.md

```diff
@@ -58,14 +58,16 @@ type LoadOutput = {
 
 If `load` returns nothing, SvelteKit will [fall through](#routing-advanced-fallthrough-routes) to other routes until something responds, or will respond with a generic 404.
 
-> `load` only applies to components that define pages, not the components that they import.
->
-> It is important to note that `load` may run on either the server or in the client browser. Code called inside `load` blocks:
->
-> - should use the SvelteKit-provided [`fetch`](#loading-input-fetch) method for getting data in order to avoid duplicate network requests
-> - should generally run on the same domain as any upstream API servers requiring credentials
-> - should not reference `window`, `document`, or any browser-specific objects
-> - should not reference any API keys or secrets directly, which will be exposed to the client, but instead call an endpoint using any required secrets
+> `load` only applies to [pages](#routing-pages) and [layouts](#layouts), and it may run on either the server or in the client browser.
+
+Code called inside `load` blocks:
+
+- should use the SvelteKit-provided [`fetch`](#loading-input-fetch) method for getting data in order to avoid duplicate network requests
```

Comment:
do we really want to take this out of the red box? this seems like the kind of thing we want to alert users to

Reply:
Leaving it inside will make it one huge red box, and when everything is an alert, nothing is. Taking this list out of it will make it so that it's placed between two red box and reads better for list items.

Reply:
The box currently (https://kit.svelte.dev/docs#loading) is just on the edge of what I'd say is the biggest we want to go. I like that you combined the first couple sentences, which will make it smaller. I am worried that adding the new stuff would push it over the edge. But I would keep what's currently there instead of removing it and adding the new stuff

Reply:
i realise this wording was already present, but the duplicate network requests thing is barely here nor there — the real reason to use the passed-in `fetch` is that nothing will work otherwise

```suggestion
- should use the SvelteKit-provided [`fetch`](#loading-input-fetch) wrapper rather than using the native `fetch`, which doesn't have access to cookies on the server and cannot make requests against the app's own endpoints
```